src/views/NewFlow.vue

```diff
@@ -26,7 +26,7 @@
 
               <a
                 :href="route('flows.editFlow', {flowId: activeFlow.uuid, component: 'builder', mode: 'edit'})"
-                class="btn btn-primary"
+                class="btn btn-primary cy--create--btn"
```

Comment:
what do you think about using `data-` attributes in the future and maybe even custom commands like `getBySel`? (see here: https://docs.cypress.io/guides/references/best-practices#Selecting-Elements)

Reply:
Good point, I forgot to check to recommendation. I think I'm gonna make changes at this PR. Thanks @bulatgab